src/httpserver/create_webserver.hpp

```diff
@@ -500,44 +252,44 @@ class create_webserver
         }
 
     private:
-        uint16_t _port;
-        http::http_utils::start_method_T _start_method;
-        int _max_threads;
-        int _max_connections;
-        int _memory_limit;
-        size_t _content_size_limit;
-        int _connection_timeout;
-        int _per_IP_connection_limit;
-        log_access_ptr _log_access;
-        log_error_ptr _log_error;
-        validator_ptr _validator;
-        unescaper_ptr _unescaper;
-        const struct sockaddr* _bind_address;
-        int _bind_socket;
-        int _max_thread_stack_size;
-        bool _use_ssl;
-        bool _use_ipv6;
-        bool _debug;
-        bool _pedantic;
-        std::string _https_mem_key;
-        std::string _https_mem_cert;
-        std::string _https_mem_trust;
-        std::string _https_priorities;
-        http::http_utils::cred_type_T _cred_type;
-        std::string _digest_auth_random;
-        int _nonce_nc_size;
-        http::http_utils::policy_T _default_policy;
-        bool _basic_auth_enabled;
-        bool _digest_auth_enabled;
-        bool _regex_checking;
-        bool _ban_system_enabled;
-        bool _post_process_enabled;
-        bool _deferred_enabled;
-        bool _single_resource;
-        bool _tcp_nodelay;
-        render_ptr _not_found_resource;
-        render_ptr _method_not_allowed_resource;
-        render_ptr _internal_error_resource;
+        uint16_t _port = DEFAULT_WS_PORT;
+        http::http_utils::start_method_T _start_method = http::http_utils::INTERNAL_SELECT;
+        int _max_threads = 0;
+        int _max_connections = 0;
+        int _memory_limit = 0;
+        size_t _content_size_limit = std::numeric_limits<size_t>::max();
```

Comment:
The change from 0 to max is definitely a correct change, but I'd rather do it in a different CR to push before pushing this.

Reply:
Mind if I do it after this one?

(Git seems to be gratuitously obtuse when it comes to dealing with merges and PRs.)